webpack/assets/javascripts/react_app/components/notifications/drawer/notificationGroup.js

```diff
@@ -39,9 +40,17 @@ export default (
               onClickedLink={onClickedLink}
               key={notification.id}
               notification={notification}
-              onMarkAsRead={onMarkAsRead.bind(this, group)}
+              onMarkAsRead={onMarkAsRead.bind(this, group, [notification.id])}
```

Comment:
@matanwerbner thats how I started, but it feels like a lot of boilerplate (nearly duplicate code) to have both mark READ as GROUP and mark READ as event..? nearly everything is doubled, the only thing that is not is the API which can change to accept an array of ids instead of a single id?